product_docs/docs/epas/16/application_programming/ecpgplus_guide/05_building_executing_dynamic_sql_statements.mdx

```diff
@@ -43,7 +43,7 @@ int main(int argc, char *argv[])
 
   EXEC SQL WHENEVER SQLERROR DO handle_error();
 
-  EXEC SQL CONNECT :argv[1];
+  EXEC SQL CONNECT TO edb;
```

Comment:
This removes the argument passing and hard wires the example to a named database.